src/frontend/src/tests/lib/utils/tokens.utils.spec.ts

```diff
@@ -156,3 +160,16 @@ describe('pinTokensWithBalanceAtTop', () => {
 		]);
 	});
 });
+
+describe('getTokensTotalUsdBalance', () => {
+	it('should correctly calculate tokens USD total balance', () => {
```

Comment:
Maybe add a test if the previded list is empty?
And another if a token has no usdBalance?

Reply:
Sounds good. Added 2 additional test cases (empty list + list with tokens w/o `usdBalance`).